numpy/core/arrayprint.py

```diff
@@ -681,14 +686,14 @@ def recurser(index, hanging_indent):
                     line += separator
 
             for i in range(trailing_items, 1, -1):
-                word = recurser(index + (-i,), next_hanging_indent)
+                word = recurser(index + (-i,), next_hanging_indent, next_width)
```

Comment:
Not strictly needed here, but in future this could be passed to the format function for printing, say, line-wrapped strings, or nested ndarrays.